docs/self-managed/zeebe-deployment/operations/update-zeebe.md

```diff
@@ -163,6 +163,10 @@ If the error is caused by `Cannot upgrade to or from a pre-release version`, Zee
 This is not permitted because pre-release versions such as alpha releases are considered unstable and do not guarantee compatibility with any other version.
 If you attempted to update from a normal version to a pre-release version, you can roll back to the previous normal version.
 
+:::note
+While in most instances version rollbacks are not supported, in this particular use case a version rollback is possible.
```

Comment:
Could we condense these two lines? Either as a single note, or as one line in the `Cannot upgrade to or from a pre-release version` paragraph, to make it clear this is referring to that error only. 

`If you attempted to update from a minor release to a pre-release or alpha version, it is possible to roll back to the previous version of Zeebe. Note that version rollbacks are not supported in most other instances.`

Reply:
Updated in latest commit.